### PR TITLE
Adding a lotta suicide acts

### PR DESCRIPTION
**Pull request by ghost**
look, things happened with the commit and i'm using the same branch it's a long story okay

suicide acts are fun to discover but there isn't enough of them

### CODE REVIEW

**Comment by Armhulen** [comment]
got dammit i'll fix that i know why too

**Comment by Armhulen** [comment]
this should be fixed

**Comment by The-clown-known-as-Pepe** [comment]
gouty if you're troubled you can speak to us

**Comment by The-clown-known-as-Pepe** [comment]
tg station is a supporting community if you have problems you can talk to us

**Comment by The-clown-known-as-Pepe** [comment]
no need to do irrational things

**Comment by DoctoreMad** [comment]
I was promised a lot of suicide acts but I got 2
Refund where

**Comment by Armhulen** [comment]
in a second if one don't work NONE OF THEM DO

**Comment by Armhulen** [comment]
also taking suggestions

File: code/modules/paperwork/pen.dm [hunk 1]
```diff
@@ -196,6 +196,13 @@
 	attack_verb = list("slashed", "stabbed", "sliced", "torn", "ripped", "diced", "cut") //these wont show up if the pen is off
 	var/on = FALSE
 
+/obj/item/pen/edagger/suicide_act(mob/living/user)
+	if(on)
+		user.visible_message("<span class='suicide'>[user] jabs [src] into their neck! It looks like [user.p_theyre()] trying to commit suicide!</span>")
```
Comment (Qustinnus) [inline code/modules/paperwork/pen.dm]: you can cancel the suicide act so this should be "is trying to jab" or else you could continuously jab yourself with the edagger :^)

File: code/modules/surgery/tools.dm [hunk 2]
```diff
@@ -8,6 +8,10 @@
 	w_class = WEIGHT_CLASS_TINY
 	origin_tech = "materials=1;biotech=1"
 
+/obj/item/retractor/suicide_act(mob/living/user)
+ 	user.visible_message("<span class='suicide'>[user] slams [src] into their eye! It looks like [user.p_theyre()] trying to commit suicide!</span>")
```
Comment (Qustinnus) [inline code/modules/surgery/tools.dm]: ditto

File: code/modules/paperwork/pen.dm (hunk 1 above)
Comment (Armhulen) [reply code/modules/paperwork/pen.dm]: suicide false is for items so horrible they shouldn't even be in the game, like SORD

File: code/modules/surgery/tools.dm (hunk 2 above)
Comment (Armhulen) [reply code/modules/surgery/tools.dm]: ditto to you too sir 🎩

**Comment by Qustinnus** [comment]
tbf almost all of them are like that, i dont know if the other suicides in general are like this but it's kinda odd if you can just spam suicide and keep cancelling it to spam messages like "john stabs himself in the head with his pen"

**Comment by Armhulen** [comment]
now hold up

**Comment by Armhulen** [comment]
how is this cancelling the suicide act

**Comment by Armhulen** [comment]
i'll have to test it later today but it shouldn't be doing that

**Comment by Supermichael777** [comment]
These are kinda toxic and should never have been added. The functional ones (grenades) need adjustment so its not skipping the timer.

**Comment by DoctoreMad** [comment]
What does "toxic" mean

File: code/modules/paperwork/pen.dm (hunk 1 above)
Comment (nicbn) [reply code/modules/paperwork/pen.dm]: Use is jabbing then

**Comment by Armhulen** [comment]
toxic????

**Comment by Armhulen** [comment]
they've already been added??????????????

**Comment by Armhulen** [comment]
grenades??????????????????????????????????????????????

**Comment by Armhulen** [comment]
i'm literally shaking...

**Comment by optimumtact** [comment]
@Supermichael777 what are you talking about?

File: code/modules/surgery/tools.dm (hunk 2 above)
Comment (optimumtact) [reply code/modules/surgery/tools.dm]: on a related note, this is why your build is failing, you've indented this with spaces and not tabs.

If you use a good text editor you can get it to show spaces and tabs differently

File: code/modules/surgery/tools.dm [hunk 3]
```diff
@@ -32,6 +35,9 @@
 	origin_tech = "materials=1;biotech=1"
 	attack_verb = list("attacked", "pinched")
 
+/obj/item/hemostat/suicide_act(mob/living/user)
+ 	user.visible_message("<span class='suicide'>[user] tears [user.p_them()]self apart with a snip, a cut, and a [src]! It looks like [user.p_theyre()] trying to commit suicide!</span>")
```
Comment (optimumtact) [inline code/modules/surgery/tools.dm]: inconsistent indentation here as well

File: code/modules/surgery/tools.dm (hunk 2 above)
Comment (Qustinnus) [reply code/modules/surgery/tools.dm]: ( https://atom.io/ )

File: code/game/objects/items/toys.dm [hunk 4]
```diff
@@ -347,6 +347,10 @@
 	attack_verb = list("attacked", "slashed", "stabbed", "sliced")
 	hitsound = 'sound/weapons/bladeslice.ogg'
 
+/obj/item/toy/katana/suicide_act(mob/living/user)
+	user.visible_message("<span class='suicide'>[user] slices the air around [user.p_them()] furiously with [src]! It looks like [user.p_theyre()] trying to unleash their final move!</span>")
+	return(BRUTELOSS|FIRELOSS)
```
Comment (ExcessiveUseOfCobblestone) [inline code/game/objects/items/toys.dm]: it's a toy so it should fail

File: code/modules/paperwork/pen.dm (hunk 1 above)
Comment (ExcessiveUseOfCobblestone) [reply code/modules/paperwork/pen.dm]: there is no suicide false here?

File: code/modules/surgery/tools.dm (hunk 3 above)
Comment (ExcessiveUseOfCobblestone) [reply code/modules/surgery/tools.dm]: p_them doesn't ever return "them" does it because themself is weird.

File: code/modules/surgery/tools.dm [hunk 5]
```diff
@@ -192,6 +212,11 @@
 	origin_tech = "biotech=1"
 	attack_verb = list("slapped")
 
+/obj/item/surgical_drapes/suicide_act(mob/living/user)
+ 	user.visible_message("<span class='suicide'>[user] drapes [src] over their head and communicates to the spooky ghost god! It looks like [user.p_theyre()] trying to ascend to the next plane!</span>")
+ 	dust(mob/user)
+	return(BRUTELOSS) //if they somehow don't get dusted i guess
```
Comment (ExcessiveUseOfCobblestone) [inline code/modules/surgery/tools.dm]: this is probably going to derp up something since it's giving bruteloss to a now null object.

**Review by ExcessiveUseOfCobblestone — COMMENTED** [review]
Bikeshed: The Review

File: code/modules/surgery/tools.dm (hunk 2 above)
Comment (optimumtact) [reply code/modules/surgery/tools.dm]: ignore that comment, the person is clearly brain damaged

Comment (Qustinnus) [reply code/modules/surgery/tools.dm]: fight me you kiwi fucker

**Comment by Armhulen** [comment]
web editor miright

**Comment by ExcessiveUseOfCobblestone** [comment]
WEBBITOR

File: code/modules/surgery/tools.dm (hunk 5 above)
Comment (ShizCalev) [reply code/modules/surgery/tools.dm]: https://github.com/tgstation/tgstation/blob/551ede6d35bc955fa83979049441321f46ab9653/code/__HELPERS/pronouns.dm#L67

it does.

File: code/modules/surgery/tools.dm (hunk 3 above)
Comment (ShizCalev) [reply code/modules/surgery/tools.dm]: https://github.com/tgstation/tgstation/blob/551ede6d35bc955fa83979049441321f46ab9653/code/__HELPERS/pronouns.dm#L67

it does.

**Comment by Cyberboss** [comment]
travis you meme

**Comment by Armhulen** [comment]
I'm sorry k I'll fix it today

**Comment by Armhulen** [comment]
actually second thought i want to focus on spiders and this pr is just to toxic sorry k

**Pull request closed without merging**